low-level/framework/utils/sysfs_interface.py

```diff
@@ -58,6 +60,13 @@ def initialize(self):
         except OSError as os_error:
             return os_error.errno
 
+    def get_sysfs_base_path(self):
+        """Returns the  sysfs base path 
+        Ex: /sys/ """
```

Comment:
fix codacy issues

Reply:
I can't see any codacy comment on the PR.
can you redirect me to the comment if any?

Reply:
Fixed codacy issue

Reply:
>low-level/framework/utils/sysfs_interface.py#L63
No blank lines allowed after function docstring (found 1) (D202)

This can be fixed

Reply:
> No blank lines allowed after function docstring (found 1) (D202)

#L63 No blank line is there after func docstring. 
This is a invalid warning